src/connectors/query-rules/connectQueryRules.ts

```diff
@@ -207,11 +212,13 @@ const connectQueryRules: QueryRulesConnector = (render, unmount = noop) => {
             hasStateRefinements(state) ||
             Boolean(widgetParams.transformRuleContexts)
           ) {
-            onHelperChange(state);
+            onHelperChange({ state });
           }
 
           // We track every change in the helper to override its state and add
           // any `ruleContexts` needed based on the `trackedFilters`.
+          // @ts-ignore
+          // @TODO: we have to update the definition to make it work
```

Comment:
We could ship the d.ts with the helper & make the modifications?

Reply:
An alternative is to update the type we use in InstantSearch: https://github.com/algolia/instantsearch.js/blob/6b64d6b0dd2153b5f40fb0121e3a19224c011420/src/types/instantsearch.ts#L109

Reply:
I didn't planed to do it in this PR at first, but make sense to do it now in fact. Will update the PR.